test/NuGet.Core.Tests/NuGet.Common.Test/DateTimeUtilityTests.cs

```diff
@@ -16,7 +16,6 @@ public static IEnumerable<object[]> GetData()
             {
                 new object[] { "1.23", TimeSpan.FromSeconds(1.23d) },
                 new object[] { "0", TimeSpan.FromMilliseconds(0.01d) }, // round down
-                new object[] { "1", TimeSpan.FromMilliseconds(0.56d) }, // round up
                 new object[] { "1", TimeSpan.FromMilliseconds(1.21d) }, // round down
```

Comment:
How the corner is affected by netcoreapp3.1 upgrade? Is it just an Assert error ?

Reply:
There's a rounding error I can't repro locally. 

It's a known consequence of https://github.com/dotnet/runtime/issues/13230, so I've been trying to figure out which test case it is.

Reply:
I'll need to dig further, I was hoping I was guessing it right, but think we'll need to do some more work.

Reply:
Fixed the test case to account for the different behavior on different frameworks.